project/BinaryIncompatibilities.scala

```diff
@@ -3,6 +3,9 @@ import com.typesafe.tools.mima.core.ProblemFilters._
 
 object BinaryIncompatibilities {
   val IR = Seq(
+    // Breaking: Utils.printEscapeJS now returns Int
+    ProblemFilters.exclude[IncompatibleResultTypeProblem](
+      "org.scalajs.core.ir.Utils.printEscapeJS")
   )
```

Comment:
According to our coding style, the indentation inside `()` is 4 spaces, not 2. So to be pedantic (because we are), this should be:
```scala
  val IR = Seq(
      // Breaking: Utils.printEscapeJS now returns Int
      ProblemFilters.exclude[IncompatibleResultTypeProblem](
          "org.scalajs.core.ir.Utils.printEscapeJS")
  )
```